src/main/java/fr/polytech/sim/log/NamedLogger.java

```diff
@@ -16,4 +19,11 @@ protected NamedLogger(String name) {
         this.name = name;
     }
 
+    @Override
+    synchronized public void log(String format, Object... args) {
+        String entry = String.format(format, args);
+        String message = String.format("%s\t%s\n", this.name, entry);
+        writeMessage(message);
+    }
+    abstract public void writeMessage(String message);
```

Comment:
Cette méthode devrait être `protected`.
```suggestion
    abstract protected void writeMessage(String message);
```